digiwf-libs/digiwf-optimize-plugins/pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>de.muenchen.oss.digiwf</groupId>
+        <artifactId>digiwf-libs</artifactId>
+        <version>1.10.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>digiwf-optimize-plugins</artifactId>
+
+    <properties>
+        <optimize.version>3.12.0</optimize.version>
+        <hamcrest.version>1.3</hamcrest.version>
```

Comment:
Gehört das nicht in die DigiWF-Test-Lib oder sonst generell in ein Parent. Hab Hamcrest in anderen Modulen auch schon vermisst bzw. teilweise wird was anderes verwendet

Reply:
In der digiwf-unit-test lib ist schon `assertj-core` vorhanden da müsste man vll insgesamt nochmal aufräumen. Hab hamcrest hier ganz entfernt, weil es gar nicht mehr verwendet wurde.

Reply:
Musste logbook raus nehmen, weil die dms Tests dann wegen fehlenden Beans fehlgeschlagen sind und ich das auf die schnelle nicht beheben konnte.